src/services/SendEmailService.ts

```diff
@@ -72,16 +72,34 @@ export class SendEmailService {
 	async sendEmail(message: Email): Promise<EmailResponse> {
     	let encoded;
     	// Fetch the instructions pdf from Yoti
-    	this.logger.debug("Fetching the Instructions Pdf from yoti for sessionId: ", message.yotiSessionId);
-    	try {
-    		const instructionsPdf = await this.yotiService.fetchInstructionsPdf(message.yotiSessionId);
-			if (instructionsPdf) {
-				encoded = Buffer.from(instructionsPdf, "binary").toString("base64");
+		let yotiInstructionsPdfRetryCount = 0;
+		//retry for maxRetry count configured value if fails
+		while (yotiInstructionsPdfRetryCount++ < this.environmentVariables.yotiInstructionsPdfMaxRetries() + 1) {
```

Comment:
could this check be simplified to while (yotiInstructionsPdfRetryCount <= this.environmentVariables.yotiInstructionsPdfMaxRetries() ) ?

Reply:
Refactored the code as discussed and tested the changes